frontend/pages/policies/PolicyPage/screens/QueryEditor.tsx

```diff
@@ -43,6 +44,7 @@ const QueryEditor = ({
   isTeamAdmin,
   isTeamMaintainer,
   isTeamObserver,
+  isObserverPlus,
```

Comment:
should this new prop be used?

Reply:
to be clear, because it's not currently being used

Reply:
Thanks for catching this--I grabbed it one component down and forgot to remove it here!